[RU] Update Beatmap/Category (#5059)
ppy/osu-wiki: wiki/Beatmap/Category/ru.md

# Категории карт

[Карты](/wiki/Beatmap), опубликованные на сайте, разделяются на категории в зависимости от даты последнего обновления, статуса [номинации](/wiki/Beatmap_ranking_procedure#квалификация) и популярности.

*Примечание: здесь и далее намеренно приводятся устоявшиеся названия категорий без перевода.*

## Существующие категории

### Graveyard

> В категорию Graveyard попадают карты, которыми больше никто не занимается. Они автоматически перемещаются сюда из категорий [Work in Progress и Pending](#work-in-progress-и-pending), если их не обновляли в течение 4 недель. У таких карт нет таблиц рекордов, и они не влияют на статистику игроков при прохождении, но их по-прежнему можно скачать и сыграть. В списке карт в клиенте они отмечены знаком вопроса (![](/wiki/shared/status/graveyard.png)).

Reviewer: ```suggestion
В категорию Graveyard попадают карты, которые были заброшены их создателем. Они автоматически перемещаются сюда из категорий [Work in Progress и Pending](#work-in-progress-и-pending), если их не обновляли в течение 4 недель. У таких карт нет таблиц рекордов, но их по-прежнему можно скачать и сыграть, плюс они влияют на [статистику при прохождении](#unranked-beatmaps-statistics). В списке карт в клиенте они отмечены знаком вопроса (![](/wiki/shared/status/graveyard.png)).
```
Первое предложение неоднозначно может быть воспринято, в оригинале всё просто. 
Теперь они влияют на статистику, и в оригинале про это написано. И не хватает ссылки  на https://osu.ppy.sh/wiki/en/Beatmap/Category#unranked-beatmaps-statistics
Только эту ссылку перевести надо, но это уже сам давай!

Author: вот что значит лениться вычитывать. спасибо

После попадания в Graveyard карту можно оживить (переместить назад в [Pending](#work-in-progress-и-pending)), обновив её через [систему загрузки карт](/wiki/Glossary#bss).

### Work in Progress и Pending

> Work in Progress (обычно сокращается до *WIP*) и Pending — категории, куда карты попадают сразу после загрузки. Отсюда их путь лежит в категорию [Qualified](#qualified) в случае [номинации](/wiki/Beatmap_ranking_procedure#номинация) как минимум двумя [номинаторами](/wiki/People/The_Team/Beatmap_Nominators). Перед этим они проходят стадию [моддинга](/wiki/Modding), когда другие пользователи в роли [моддеров](/wiki/Glossary#modder) вносят свои замечания и предложения по улучшению. У карт в Work in Progress и Pending нет таблиц рекордов, и они также не влияют на статистику игроков при прохождении. В списке карт в клиенте они отмечены знаком вопроса (![](/wiki/shared/status/pending.png)).

Author: ```suggestion
Work in Progress (обычно сокращается до *WIP*) и Pending — категории, куда карты попадают сразу после загрузки. Отсюда их путь лежит в категорию [Qualified](#qualified) в случае [номинации](/wiki/Beatmap_ranking_procedure#номинация) как минимум двумя [номинаторами](/wiki/People/The_Team/Beatmap_Nominators). Перед этим они проходят стадию [моддинга](/wiki/Modding), когда другие пользователи в роли [моддеров](/wiki/Glossary#modder) вносят свои замечания и предложения по улучшению. У карт в Work in Progress и Pending нет таблиц рекордов, но при прохождении они влияют на [игровую статистику](#статистика-неранкнутых-карт). В списке карт в клиенте они отмечены знаком вопроса (![](/wiki/shared/status/pending.png)).
```

Карты, которые не обновлялись в течение 4 недель, автоматически перемещаются в категорию [Graveyard](#graveyard).

### Qualified

> Категория Qualified — последний этап [процедуры ранкинга](/wiki/Beatmap_ranking_procedure). Карты, находящиеся здесь, проверены как минимум двумя [номинаторами](/wiki/People/The_Team/Beatmap_Nominators) на соответствие [критериям ранкинга](/wiki/Ranking_Criteria). При попадании в Qualified карта также встаёт в [очередь ранкинга](/wiki/Beatmap_ranking_procedure/Ranking_queue) и в конечном счёте попадает в категорию [Ranked](#ranked) при условии, что её не [дисквалифицируют](/wiki/Beatmap_ranking_procedure#сброс-номинации). Это — последний этап перед ранком, в течение которого все серьёзные недостатки и ошибки карты, упущенные моддерами и номинаторами, должны быть исправлены.

Reviewer: ```suggestion
Категория Qualified — последний этап [процедуры ранкинга](/wiki/Beatmap_ranking_procedure). Карты, находящиеся здесь, проверены как минимум двумя [номинаторами](/wiki/People/The_Team/Beatmap_Nominators) на соответствие [критериям ранкинга](/wiki/Ranking_Criteria). При попадании в Qualified карта также встаёт в [очередь ранкинга](/wiki/Beatmap_ranking_procedure/Ranking_queue) и в конечном счёте попадает в категорию [Ranked](#ranked) при условии, что её не [дисквалифицируют](/wiki/Beatmap_ranking_procedure#сброс-номинации). Это последний этап перед ранком, в течение которого все недостатки и ошибки карты, упущенные моддерами и номинаторами, должны быть исправлены.
```
Тире тут смущает.
И переврал, в оригинале «any flaws and mistakes», а у тебя стало «серьёзным».

Игроки не получают [очки производительности](/wiki/Performance_points) за прохождение карт из категории Qualified, и хотя у них есть таблицы рекордов, они будут сброшены при перемещении карт в другую категорию. В списке карт в клиенте они отмечены галочкой (![](/wiki/shared/status/qualified.png)).

### Ranked

Категория Ranked содержит карты, считающиеся «официальными» в смысле соответствия [критериям ранкинга](/wiki/Ranking_Criteria). У них есть таблицы рекордов, где игроки соревнуются за первенство и получают [очки производительности](/wiki/Performance_points) за достижение рекордов. Карты, находящиеся в категории Ranked, убираются оттуда только в исключительных случаях. В списке карт в клиенте они отмечены двойным [шевроном](https://ru.wikipedia.org/wiki/Шеврон_(знак_различия) "Wikipedia"), похожим на двойную стрелку вверх (![](/wiki/shared/status/ranked.png)).

### Loved

*Подробнее: [История категории Loved](/wiki/Beatmaps/History_of_Loved)*

> Карты категории Loved почти всегда нарушают [критерии ранкинга](/wiki/Ranking_Criteria), но, тем не менее, игровое сообщество их *любит* и каждый месяц пополняет эту категорию в рамках [Project Loved](/wiki/Project_Loved). У таких карт есть таблицы рекордов, но они не приносят [очков производительности](/wiki/Performance_points) за прохождение, более того, все рекорды обнуляются при убирании карт из Loved (что обычно случается только по просьбе их авторов). В списке карт в клиенте они отмечены сердцем (![](/wiki/shared/status/loved.png)).

Reviewer: ```suggestion
Карты категории Loved могут нарушать [критерии ранкинга](/wiki/Ranking_Criteria), но, тем не менее, игровое сообщество их *любит* и каждый месяц пополняет эту категорию в рамках [Project Loved](/wiki/Project_Loved). У таких карт есть таблицы рекордов, но они не приносят [очков производительности](/wiki/Performance_points) за прохождение, более того, все рекорды обнуляются при убирании карт из Loved (что обычно случается только по просьбе их создателей). В списке карт в клиенте они отмечены сердцем (![](/wiki/shared/status/loved.png)).
```
Снова переврал. Оригинал: "**may** not meet the ranking criteria"
У тебя же "почти всегда нарушают". 
«Автор» мне не нравится по отношению к мапперам.

> Карты категории Loved могут нарушать [критерии ранкинга](/wiki/Ranking_Criteria), но, тем не менее, игровое сообщество их *любит* и каждый месяц пополняет эту категорию в рамках [Project Loved](/wiki/Project_Loved). У таких карт есть таблицы рекордов, но они не приносят [очков производительности](/wiki/Performance_points) за прохождение, более того, все рекорды обнуляются при убирании карт из Loved (что обычно случается только по просьбе их создателей). В списке карт в клиенте они отмечены сердцем (![](/wiki/shared/status/loved.png)).

Reviewer: ```suggestion
Карты категории Loved могут нарушать [критерии ранкинга](/wiki/Ranking_Criteria), но, тем не менее, игровое сообщество их *любит* и каждый месяц пополняет эту категорию в рамках [Project Loved](/wiki/Project_Loved). У таких карт есть таблицы рекордов, но они не приносят [очков производительности](/wiki/Performance_points) за прохождение, более того, все рекорды обнуляются при удалении карт из Loved (что обычно случается только по просьбе их создателей). В списке карт в клиенте они отмечены сердцем (![](/wiki/shared/status/loved.png)).
```
«убирание» слишком разговорно звучит.

## История

### Approved

Категория Approved использовалась с 2008 по 2014 годы и содержала карты, которые нельзя было ранкнуть из-за их длины, максимально возможного числа очков или иного нарушения старых версий [критериев ранкинга](/wiki/Ranking_Criteria).

Такие карты начали влиять на ранк игроков после того, как система ранкинга стала основываться на [очках производительности](/wiki/Performance_points). Затем категория Approved стала использоваться только для карт со [временем игры](/wiki/Gameplay/Drain_time) больше 6 минут.

Карты категории Approved отмечены галочкой (![](/wiki/shared/status/approved.png)) в списке карт в клиенте и имеют такую же игровую механику (таблицы рекордов и влияние на статистику), как и [ранкнутые карты](#ranked).

### Появление Qualified

> Категория [Qualified](#qualified) была добавлена в августе 2014 года в рамках [реформирования BAT](https://osu.ppy.sh/home/news/2014-08-21-restructuring-of-the-bat), чтобы только что ранкнутые карты были более заметны. Если такую карту дисквалифицировали и перемещали назад в Pending, это никак не отражалось на статистике игроков: Qualified с самого начала задумывалась как временная категория со временными рекордами. Это позволило игрокам сформировать правильные ожидания, а также сильно уменьшило их негативное отношение к анранку карт.

Reviewer: ```suggestion
Категория [Qualified](#qualified) была добавлена в августе 2014 года в рамках [реформирования BAT](https://osu.ppy.sh/home/news/2014-08-21-restructuring-of-the-bat), чтобы только что ранкнутые карты были более заметны. Если такую карту дисквалифицировали и перемещали назад в Pending, это никак не отражалось на статистике игроков: Qualified с самого начала задумывалась как категория с временными рекордами. Такой метод позволял игрокам не рассчитывать на гарантированный ранк, а также сильно уменьшил их негативное отношение к анранку карт.
```
Два раза «временно» не нужно, да и категория-то не временная, она уже столько лет с нами.
«сформировать правильные ожидания» звучит не по-русски. Оригинальное «set correct expectations» в моём варианте раскрыл.

Author: чуть подправил (метод -> подход) и принял

> ### Неранкнутые карты и число очков

Reviewer: ```suggestion
### Статистика неранкнутых карт
```
Так как плейкаунт ещё дают, а не только очки. Да и в целом непонятно, что за число очков.

> Начиная с [7 августа 2020 года](https://osu.ppy.sh/home/changelog/stable40/20200807.3), карты из категории Graveyard, как и [Work in Progress, и Pending](#work-in-progress-и-pending), стали влиять на число игр пользователя и его общее число очков. Помимо этого, на странице таких карт можно посмотреть игровую статистику (число игр, процент успешных прохождений и наиболее сложные участки), до этого присущую только картам из категорий [Qualified](#qualified) и [Ranked](#ranked).

Reviewer: ```suggestion
Начиная с [7 августа 2020 года](https://osu.ppy.sh/home/changelog/stable40/20200807.3), карты из категории [Graveyard](#graveyard), как и [Work in Progress, и Pending](#work-in-progress-и-pending), стали влиять на число игр пользователя и его общее число очков. Помимо этого, на странице таких карт можно посмотреть игровую статистику (число игр, процент успешных прохождений и наиболее сложные участки), до этого присущую только картам из категорий [Qualified](#qualified) и [Ranked](#ranked).
```
